CHANGES.md

```diff
@@ -10,6 +10,7 @@
 
 <!-- Changes that affect Black's stable style -->
 - Fix crash while formatting a long `del` statement containing tuples (#4628)
+- Fix crash while formatting names expressions in complex `with` statements (#4630)
```

Comment:
```suggestion
- Fix crash while formatting expressions using the walrus operator in complex `with` statements (#4630)
```
I think you meant "named" instead of "names" expressions. That's the formal name but I think "walrus" will be more recognizable to most readers. (Might need reformatting for line length)